### PR TITLE
docs: add org README

### PR DESCRIPTION
**Pull request by nikolasburk**
Adding an org README, see more context: 

- [internal Slack](https://prisma-company.slack.com/archives/C02UA0D53KM/p1657186120655939)
- [Notion](https://www.notion.so/prismaio/Prisma-GitHub-Org-README-2a4c59055aa2488ab57f32659d033c5a)

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <br />
+  <a href="https://www.prisma.io/docs/getting-started/quickstart">Quickstart</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://www.prisma.io/">Website</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://www.prisma.io/docs/">Docs</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://github.com/prisma/prisma-examples/">Examples</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://prisma.io/data-platform">Prisma Data Platform</a>
```
Comment (Jolg42) [inline profile/README.md:15]:
```suggestion
  <a href="https://www.prisma.io/data-platform">Prisma Data Platform</a>
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  <br />
+  <hr />
+</div>
+
+## 👋 Welcome
+
+Prisma makes working with databases easy! Currently, we are building two main tools:
+
+- [Prisma ORM](https://github.com/prisma/prisma) (Node.js & TypeScript | PostgreSQL, MySQL, MariaDB, SQL Server, SQLite, CockroachDB & MongoDB)
+- [Prisma Data Platform](https://prisma.io/data-platform) for development teams and organizations
```
Comment (Jolg42) [inline profile/README.md:25]:
```suggestion
- [Prisma Data Platform](https://www.prisma.io/data-platform) for development teams and organizations
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+You can learn about upcoming Prisma features on the [roadmap](https://pris.ly/roadmap) and ask questions via [GitHub Discussions](https://github.com/prisma/prisma/discussions). The Prisma ORM is released every three weeks, you can learn more in the [release notes](https://github.com/prisma/prisma/releases).
+
+
+## 🎁 Repositories
+
+This GitHub organization contains repos that are _officially_ maintained by [Prisma](https://prisma.io):
```
Comment (Jolg42) [inline profile/README.md:32]:
```suggestion
This GitHub organization contains repos that are _officially_ maintained by [Prisma](https://www.prisma.io):
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- The Prisma ORM can be found in the main [`prisma/prisma`](https://github.com/prisma/prisma) repo. 
+- The [documentation](https://prisma.io/docs) for the ORM and the Prisma Data Platform is in located the [`prisma/docs`](https://github.com/prisma/docs) repo.
```
Comment (Jolg42) [inline profile/README.md:35]:
```suggestion
- The [documentation](https://www.prisma.io/docs) for the ORM and the Prisma Data Platform is in located the [`prisma/docs`](https://github.com/prisma/docs) repo.
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## 💚 Community
+
+Please join the [Prisma Community](https://prisma.io/community), we're happy to have you here!
```
Comment (Jolg42) [inline profile/README.md:39]:
```suggestion
Please join the [Prisma Community](https://www.prisma.io/community), we're happy to have you here! 
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- [Check out our events](https://www.prisma.io/events), we are regularly hosting Meetups, conferences, livestreams — online & in-person around the globe
+- [Ask a question](https://github.com/prisma/prisma/discussions) on GitHub Discussions
+- [Report a bug or submit a feature request](https://github.com/prisma/prisma/issues) on GitHub
+
+
+### 📚 Resources
+
+You can learn more about Prisma and its rich ecosystem and community with these resources:
+
+- [Prisma Blog](https://prisma.io/blog): Announcements, tutorials, in-depth articles & more
```
Comment (Jolg42) [inline profile/README.md:58]:
```suggestion
- [Prisma Blog](https://www.prisma.io/blog): Announcements, tutorials, in-depth articles & more
```

File: profile/README.md (hunk 2 above)
Comment (janpio) [inline profile/README.md:25]: here as well:
```suggestion
- [Prisma Data Platform](https://www.prisma.io/data-platform) for development teams and organizations
```

File: profile/README.md (hunk 3 above)
Comment (janpio) [inline profile/README.md:32]:
```suggestion
This GitHub organization contains repos that are _officially_ maintained by [Prisma](https://www.prisma.io):
```

File: profile/README.md (hunk 5 above)
Comment (janpio) [inline profile/README.md:39]:
```suggestion
Please join the [Prisma Community](https://www.prisma.io/community), we're happy to have you here! 
```

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+- [YouTube](https://www.youtube.com/watch?v=acvjE2EpMbs&ab_channel=Prisma): Video walkthroughs, courses, livestreams, talks & workshops, ... 
+- [What's New in Prisma](https://www.youtube.com/playlist?list=PLn2e1F9Rfr6l1B9RP0A9NdX7i7QIWfBa7): Regular livestream with news about the Prisma ecosystem & community
+- [Prisma Day](https://www.prisma.io/day): Our yearly conference where the Prisma Community gets together
+- [Success Stories](https://prisma.io/showcase): Learn how people successfully use Prisma in production
```
Comment (janpio) [inline profile/README.md:62]:
```suggestion
- [Prisma Blog](https://www.prisma.io/blog/): Announcements, tutorials, in-depth articles & more
- [YouTube](https://www.youtube.com/watch?v=acvjE2EpMbs&ab_channel=Prisma): Video walkthroughs, courses, livestreams, talks & workshops, ... 
- [What's New in Prisma](https://www.youtube.com/playlist?list=PLn2e1F9Rfr6l1B9RP0A9NdX7i7QIWfBa7): Regular livestream with news about the Prisma ecosystem & community
- [Prisma Day](https://www.prisma.io/day): Our yearly conference where the Prisma Community gets together
- [Success Stories](https://www.prisma.io/showcase): Learn how people successfully use Prisma in production
```

**Review by Jolg42 — APPROVED** [review]
See suggestions, we should only use the `www.` version for all things `prisma.io`

**Pull request merged**